providers/implementations/include/prov/names.h

```diff
@@ -233,6 +233,10 @@
 
 #define PROV_NAMES_SHAKE_128 "SHAKE-128:SHAKE128:2.16.840.1.101.3.4.2.11"
 #define PROV_NAMES_SHAKE_256 "SHAKE-256:SHAKE256:2.16.840.1.101.3.4.2.12"
+#define PROV_NAMES_SHAKE_128_128 "SHAKE-128/128:SHAKE128/128"
+#define PROV_NAMES_SHAKE_256_256 "SHAKE-256/256:SHAKE256/256"
+#define PROV_NAMES_SHAKE_128_256 "SHAKE-128/256:SHAKE128/256"
+#define PROV_NAMES_SHAKE_256_512 "SHAKE-256/512:SHAKE256/512"
```

Comment:
Are we clear on what those who fetch by OID may expect?

Reply:
Yeah, this is to be decided yet, whether we want to move the OIDs to the _proper default length_ variants.

Reply:
We know that *TLS* expects the higher defaults...  but on the other hand, TLS doesn't play with OIDs for cipher suite selection, so...

Reply:
It was CMS (and CMP and certificates).

Reply:
> @levitte sorry for barging in, but why you're mentioning TLS? AFAIK, TLS doesn't use SHAKE for anything (the only thing I can think of is Ed448, but that's internal to it, not directly accessed)

I was misremembering and didn't go back and check.  Sorry.  @t8m got it right